utilities/rocAL/rocAL_unittests/testAllScripts.sh

```diff
@@ -26,7 +26,7 @@ caffe2_classification_path=${ROCAL_DATA_PATH}/rocal_data/caffe2/classification/
 caffe2_detection_path=${ROCAL_DATA_PATH}/rocal_data/caffe2/detection/
 mxnet_path=${ROCAL_DATA_PATH}/rocal_data/mxnet/
 output_path=../rocal_unittest_output_folder_$(date +%Y-%m-%d_%H-%M-%S)/
-golden_output_path=${ROCAL_DATA_PATH}/rocal_data/GoldenOutputs/
+golden_output_path=${ROCAL_DATA_PATH}/rocal_data/GoldenOutputsTensor/
```

Comment:
Do we have 2 golden output folders for batchpd and tensor?

Reply:
Yes